mlrun/utils/helpers.py

```diff
@@ -667,6 +674,29 @@ def enrich_image_url(image_url: str, client_version: str = None) -> str:
     return image_url
 
 
+def resolve_image_tag_suffix(
+    mlrun_version: str = None, python_version: str = None
+) -> str:
+    """
+    resolves what suffix should be appended to the image tag
+    :param mlrun_version: the mlrun version
+    :param python_version: the requested python version
+    :return: the suffix to append to the image tag
+    """
+    if not python_version or not mlrun_version:
+        return ""
```

Comment:
no, because that means that older client is calling the API which wasn't passing that information to the server, meaning it python3.7 version was the default one without the -py37 prefix